WordPress/Classes/ViewRelated/Menus/MenuItemView.m

```diff
@@ -64,14 +64,20 @@ - (void)setupCancelButton
     button.titleLabel.adjustsFontForContentSizeCategory = YES;
     [button setTitle:NSLocalizedString(@"Cancel", @"") forState:UIControlStateNormal];
 
-    UIEdgeInsets inset = button.contentEdgeInsets;
-    inset.left = 6.0;
-    inset.right = inset.left;
-    button.contentEdgeInsets = inset;
+    if (@available(iOS 15, *)) {
+        UIButtonConfiguration *configuration = [UIButtonConfiguration plainButtonConfiguration];
+        configuration.contentInsets = NSDirectionalEdgeInsetsMake(0, 6, 0, 6);
+        button.configuration = configuration;
+    } else {
+        UIEdgeInsets inset = button.contentEdgeInsets;
+        inset.left = 6.0;
+        inset.right = inset.left;
+        button.contentEdgeInsets = inset;
+    }
     button.hidden = YES;
 
     [self.accessoryStackView addArrangedSubview:button];
-    [button setContentHuggingPriority:UILayoutPriorityDefaultHigh forAxis:UILayoutConstraintAxisHorizontal];
+    [button setContentHuggingPriority:UILayoutPriorityRequired forAxis:UILayoutConstraintAxisHorizontal];
```

Comment:
Not sure why hugging behavior changes with the new button configuration. I couldn't find any related documentation. I asked on Mastodon, and other folks also encountered it. The goal is to ensure the label on the right has a lower hugging priority.